### PR TITLE
[WIP] button: improve JAWS readability 

### LINKED ISSUE #1479: button: improve JAWS readability
JAWS should read
- [x] `button` as "button"
- [x] `button_type_link` as "link"
- [x] `button_type_submit` as "button"
- [x] `button_togglable_check` as "toggle button"
- [x] `button_togglable_radio` as "toggle button"
- [x] `_disabled` as "unavailable"
- [x] `button_checked` with specified `_togglable` modifier as "toggle button pressed"
- [x] `text` special field
- [x] `title` special field

Check and fix divergencies.

### PR DESCRIPTION
**Pull request by sipayRT**
Closes #1479

### CODE REVIEW

**Comment by sipayRT** [comment]
@aristov need your review

**Comment by sipayRT** [comment]
@aristov need your review

File: common.blocks/button/_togglable/button_togglable.bh.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module.exports = function(bh) {
+    bh.match(['button_togglable_check', 'button_togglable_radio'], function(ctx) {
+        ctx
+            .applyBase()
+            .attr('aria-pressed', ctx.mods().checked || false);
```
Comment (aristov) [inline common.blocks/button/_togglable/button_togglable.bh.js]: Why not `ctx.mod('checked')`?

File: common.blocks/button/_togglable/button_togglable.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+/**
+ * @exports
+ * @class button
+ * @bem
+ */
+provide(Button.decl({ modName : 'togglable' }, /** @lends button.prototype */{
+    onSetMod : {
+        'checked' : function(_, modVal) {
+            this.__base.apply(this, arguments);
+
```
Comment (aristov) [inline common.blocks/button/_togglable/button_togglable.js]: As [I see](https://github.com/bem/bem-components/blob/v2/common.blocks/button/button.js#L34) in our codebase, this empty line is redundant.

File: common.blocks/checkbox/_type/checkbox_type_button.deps.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [{
-    mustDeps : 'button'
+    shouldDeps : { block : 'button', mods : { togglable : 'check' } }
```
Comment (aristov) [inline common.blocks/checkbox/_type/checkbox_type_button.deps.js:2]: Can we do this by separate PR, or it causes wrong `tmpl-specs` processing?

File: common.blocks/button/_type/button_type_link.js [hunk 4]
```diff
@@ -23,12 +23,17 @@ provide(Button.decl({ modName : 'type', modVal : 'link' }, /** @lends button.pro
         'disabled' : {
             'true' : function() {
                 this.__base.apply(this, arguments);
-                this.domElem.removeAttr('href');
+
```
Comment (aristov) [inline common.blocks/button/_type/button_type_link.js]: Redundant line break.

File: common.blocks/checkbox/_type/checkbox_type_button.deps.js (hunk 3 above)
Comment (sipayRT) [inline common.blocks/checkbox/_type/checkbox_type_button.deps.js:2]: Tests will fail without this changes

**Comment by sipayRT** [comment]
something wrong with Travis. The same problem in another [PR](https://github.com/bem/bem-components/pull/1551)

**Comment by sipayRT** [comment]
https://github.com/bem/bem-components/pull/1553
